scripts/20_save_data.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/sh
```

Comment:
Why did you choose to name the file save_data, i think backup much better describes the intended behaviour of this file

Reply:
i did have it backup first, but I explicitly decided NOT to name it backup, as we are not actually backing up anything. After a reboot the 'backup' is gone. So I wanted to make sure it did not give a sense of having a backup for later usage.

But i"m very much open to better naming then backup.
(I borrowed the name save_data from openwrt where they also save the cfg data between updates)